docs/custom.css

```
@@ -32,6 +32,10 @@
 @import 'tailwindcss' source('./src/');
 @import 'tw-animate-css';
 
+/* Scan the Algolia search plugin's compiled output so its Tailwind utility classes are generated.
+   The plugin's index.css contains `@source "./dist/"` resolved relative to the package. */
+@import '@mastra/docusaurus-plugin-algolia';
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)